### ISSUE TITLE
Test failures that exceed the default bytes limit aren't uploaded to artifacts

### ISSUE THREAD

**Issue by keith** [bazelbuild/continuous-integration#1215]
This failure https://buildkite.com/bazel/rules-foreign-cc/builds/3456#75a81178-883e-4acb-8548-de5300f827e5 has no artifacts and no output in the log because of the default limit

**Comment by keith** [comment]
Ah maybe this wasn't a test failure and that's why?

**Comment by philwo** [comment]
Yeah, indeed! That was a build failure, so there are no artifacts to upload.

**Comment by keith** [comment]
Think we could capture the stdout log referenced in that job in this case? Kinda leaves you no where to debug otherwise
